contracts/FlowIDTableStaking.cdc

```diff
@@ -815,6 +824,36 @@ pub contract FlowIDTableStaking {
             FlowIDTableStaking.account.save(false, to: /storage/stakingEnabled)
         }
 
+        access(account) fun removeAndRefundNodeRecord(nodeRecord: &NodeRecord) {
```

Comment:
Can you add some comments above this function?

Reply:
Will do.